install.sh

```diff
@@ -95,16 +94,14 @@ nvidia-modprobe -u -c0
 
 # configure persistence daemon
 # decreases latency for later driver loads
-# reduces nvidia-smi invocation time 10x from 30 to 2 sec 
+# reduces nvidia-smi invocation time 10x from 30 to 2 sec
 # notable on large VM sizes with multiple GPUs
 # especially when nvidia-smi process is in CPU cgroup
 cp /opt/gpu/nvidia-persistenced.service /etc/systemd/system/nvidia-persistenced.service
 systemctl enable nvidia-persistenced.service
 systemctl restart nvidia-persistenced.service
 nvidia-smi
 
-cp -r  /opt/gpu/nvidia-docker2_${NVIDIA_DOCKER_VERSION}/* /usr/
```

Comment:
Question: What is the intent of this command? Is this to update / overwrwite the `/etc/docker/daemon.json` file so that this includes the `nvidia` runtime? If this is the case, then we would need to replace this with a command to create the config or update it in place.

For example:
```
nvidia-ctk runtime configure --runtime=docker
```

Reply:
AKS nodes use containerd as the default. AKS clusters with much older k8s minor versions use docker still. This may be for accommodating those workloads - though I'm not sure.